Formula/dvc.rb

```diff
@@ -366,6 +405,11 @@ class Dvc < Formula
     sha256 "f28eac771ec9eb4866d3fb4ab65abd42d38c424739e80c08d8d20570de60b0ef"
   end
 
+  resource "hydra-core" do
+    url "https://github.com/facebookresearch/hydra/archive/refs/tags/v1.2.0.tar.gz"
+    sha256 "19b203fc614426cd6e4bb7c51e73a25a1ceb4606450ec0203345aec67a0a4f6a"
+  end
```

Comment:
Sadly will break `bump-formula-pr`/`update-python-resources`. One workaround would be to remove this resource and then run command. If upstream hasn't fixed PyPI tarball (https://github.com/facebookresearch/hydra/issues/2024), then would need to replace PyPI resource with GitHub

Reply:
I'd be open to adding this is a formula, since it's a bit annoying to use as a `resource`.

Reply:
At least for this PR, may be better to leave as is given we delayed update for a while.

First thing may be to see if `hydra-core` is willing to add all files needed to source build into PyPI tarball. I think it should be a simple change in MANIFEST.in. Seems a bit odd that they provide source code that can't be built.

Otherwise, can provide formula, but I think it will be a library installed inside virtualenv making it less useful. Currently only MacPorts and AUR package it https://repology.org/project/python:hydra-core/versions

Reply:
Yes, we can leave it this way for now.